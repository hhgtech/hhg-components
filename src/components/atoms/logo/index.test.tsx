import React from 'react'
import { render, cleanup, screen } from '@testing-library/react'
import 'jest-styled-components'

import { Logo } from '.'

afterEach(cleanup)

test('it renders', () => {
  const logo = render(<Logo type="hellobacsi" width="100px" height="auto" />)

  const self = logo.container.firstChild

  expect(self).toMatchSnapshot()

  expect('svg').toBeInTheDocument
})
