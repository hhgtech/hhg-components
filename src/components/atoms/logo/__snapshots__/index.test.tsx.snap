// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div>
  <svg
    height="auto"
    viewBox="0 0 121 23.84"
    width="100px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <defs>
      <clippath
        id="clip-path"
        transform="translate(0 -5.81)"
      >
        <path
          d="M0 0H121V32H0z"
          fill="none"
        />
      </clippath>
    </defs>
    <g
      data-name="Layer 2"
    >
      <g
        clip-path="url(#clip-path)"
        data-name="Layer 1"
        fill-rule="evenodd"
      >
        <path
          d="M107.16 6a11 11 0 014.11.89 4.42 4.42 0 01.93.56 2.42 2.42 0 01.64.71 1.42 1.42 0 01-1.15 2.08 1.21 1.21 0 01-.67-.13c-.16-.1-.33-.19-.48-.3a4.71 4.71 0 00-2.48-.95 6.57 6.57 0 00-2.21.1 2.19 2.19 0 00-.89.37 1.54 1.54 0 00-.71 1.27 1.85 1.85 0 00.73 1.63 4.6 4.6 0 00.93.56 14.1 14.1 0 001.89.66c.66.22 1.36.42 2 .68a9.52 9.52 0 011.43.71 4.59 4.59 0 012.27 3.92 6.61 6.61 0 01-.16 1.93 4.29 4.29 0 01-1.92 2.62 6.72 6.72 0 01-2.26.85 9.92 9.92 0 01-2.18.17 9.76 9.76 0 01-4.3-1.13 5.38 5.38 0 01-1-.71 2.27 2.27 0 01-.32-.3 1.16 1.16 0 01-.18-1.29 1.49 1.49 0 011.28-.92 1.11 1.11 0 01.64.13l.38.23a6.09 6.09 0 003.52 1.22 7.25 7.25 0 002-.18l.3-.1a1.94 1.94 0 001.38-1.73 2.33 2.33 0 00-1-2.29 5.7 5.7 0 00-1-.56 18.5 18.5 0 00-2-.69 18.37 18.37 0 01-2-.68 6.88 6.88 0 01-1.46-.79 4.13 4.13 0 01-1.77-3.08 4.67 4.67 0 01.09-1.75 4 4 0 011.7-2.51 6.08 6.08 0 012.67-1 3.35 3.35 0 011.25-.2zM86.32 15a13.36 13.36 0 01.61-4A8.43 8.43 0 0188 8.7a6 6 0 014.14-2.55 8.41 8.41 0 013.46.14 6 6 0 013.43 2.4l.07.11a1.34 1.34 0 01-.9 2.1 1.32 1.32 0 01-1.45-.56c-.11-.13-.19-.26-.29-.37A3 3 0 0095 9a5.55 5.55 0 00-2.51 0 3.53 3.53 0 00-2.38 1.75 6.17 6.17 0 00-.7 2 14.78 14.78 0 00-.19 3.25 10.41 10.41 0 00.3 2.14 5.5 5.5 0 00.63 1.6 3.59 3.59 0 002.93 1.8 5.27 5.27 0 001.68-.09 3.29 3.29 0 001.95-1.2c.1-.13.19-.26.3-.38a1.27 1.27 0 011.63-.24 1.4 1.4 0 01.45 2.12A5.91 5.91 0 0196 24a7.89 7.89 0 01-4.9 0 5.79 5.79 0 01-3.2-2.48 8.55 8.55 0 01-1.2-3 13.4 13.4 0 01-.38-3.52zM118.14 23.92h-1.44a1.39 1.39 0 01-1.17-2.07 1.25 1.25 0 01.74-.63l.33-.1a.2.2 0 00.15-.19V9.49c0-.17-.06-.21-.23-.27l-.2-.06a1.36 1.36 0 01-1-1.53 1.32 1.32 0 011.33-1.15h3A1.31 1.31 0 01121 7.64a1.39 1.39 0 01-.9 1.51 1.8 1.8 0 01-.27.07c-.17 0-.23.14-.24.31v11.19c0 .36 0 .37.37.46a1.4 1.4 0 01.89 2 1.28 1.28 0 01-1.16.72zM64.27 21.07a3.07 3.07 0 01-.52.06h-3.41a1.36 1.36 0 01-.75-.2 1.11 1.11 0 01-.5-.77 2.53 2.53 0 010-.38v-2.45-.3A1.15 1.15 0 0160.23 16h3.64a2.54 2.54 0 012.44 1.88 2.5 2.5 0 01-2.04 3.19zm-5.2-10.63a1.18 1.18 0 011.27-1.18H63.31a2 2 0 01.16 3.92 3.1 3.1 0 01-.43 0h-2.75A1.17 1.17 0 0159.06 12c0-.53 0-1 .01-1.56zm8.23 4c-.09-.08-.18-.14-.31-.24l.36-.53a4.48 4.48 0 00-.08-5.05 4.58 4.58 0 00-4-2.14H57.58a1.46 1.46 0 00-1.38 1.35V22.58a1.44 1.44 0 00.8 1.19 1.34 1.34 0 00.77.16h6.16a5.12 5.12 0 00.87-.08 5.37 5.37 0 004.48-4.91 5.09 5.09 0 00-1.98-4.5zM79.82 17.67a1.51 1.51 0 01-.3 0h-3.3c-.58 0-.8-.32-.6-.83.64-1.56 1.28-3.09 1.92-4.64a1.14 1.14 0 01.1-.19.29.29 0 01.48 0 .86.86 0 01.13.23l1.91 4.59a.8.8 0 01.07.2.49.49 0 01-.41.64zm5.64 4.47l-.09-.2c-.88-2.13-1.75-4.22-2.64-6.35-1.12-2.67-2.22-5.35-3.33-8a3.51 3.51 0 00-.33-.64 1.36 1.36 0 00-2.28 0 4 4 0 00-.38.67l-2.15 5.16-3.75 9a2 2 0 00-.18.57 1.43 1.43 0 001 1.55 1.3 1.3 0 001.6-.7c.1-.2.17-.42.27-.63.18-.42.37-.86.54-1.28a1.21 1.21 0 011.15-.74H80.79a1.2 1.2 0 011.21.65 1.66 1.66 0 01.13.32q.28.7.6 1.44a1.88 1.88 0 00.24.44 1.38 1.38 0 001.19.55 1.59 1.59 0 001.15-.66 1.12 1.12 0 00.15-1.15z"
          fill="#284a75"
          transform="translate(0 -5.81)"
        />
        <path
          d="M0 14.85V7.08a1.34 1.34 0 011.53-1.26 1.28 1.28 0 011.25 1 1.79 1.79 0 010 .47v3.42a.35.35 0 00.43.27l.33-.12a5.7 5.7 0 016.15 1.37 5.32 5.32 0 011.57 3.82v6.6a1.25 1.25 0 01-1 1.2 1.39 1.39 0 01-1.76-.93 1.77 1.77 0 010-.52v-6.29a2.68 2.68 0 00-.5-1.65 2.89 2.89 0 00-4-.63A2.73 2.73 0 002.83 16v6.34a1.77 1.77 0 01-.1.72 1.36 1.36 0 01-1.48.87A1.35 1.35 0 010 22.74a3 3 0 010-.52v-7.37zM38.63 15.2v6.52A2.52 2.52 0 0136 23.93a1.31 1.31 0 01-1.3-1.25 1.29 1.29 0 01.82-1.42.35.35 0 00.25-.39V9.63a.45.45 0 00-.3-.5 1.4 1.4 0 01-.6-2A1.31 1.31 0 0136 6.48a2.39 2.39 0 012.24 1.16 2.08 2.08 0 01.37 1v.6c.03 2.02.02 4 .02 5.96zM28.51 15.18V9a2.4 2.4 0 01.55-1.6 2.46 2.46 0 012-.94 1.34 1.34 0 011.31 1 1.45 1.45 0 01-.59 1.59l-.19.1a.36.36 0 00-.26.36v11.13a.54.54 0 00.37.64 1.34 1.34 0 01.73 1.39 1.36 1.36 0 01-1 1.18 2.08 2.08 0 01-1.28-.12 2.54 2.54 0 01-1.64-2.34c.02-2.05 0-4.12 0-6.21z"
          fill="#2d87f3"
          transform="translate(0 -5.81)"
        />
        <path
          d="M47.47 24.35a5.34 5.34 0 01-1.54-.15.38.38 0 00-.24 0 3.14 3.14 0 01-2 2.49 6.26 6.26 0 01-1.41.38 13.28 13.28 0 01-4 0 6 6 0 01-1.68-.51 3.09 3.09 0 01-1.6-1.62.75.75 0 00-.09-.15 3 3 0 00-.19-.74 1.2 1.2 0 00-2.3.5 2.28 2.28 0 00.22.95 1.17 1.17 0 00.05.2 4.5 4.5 0 00.72 1.3 5.71 5.71 0 002.37 1.8 9.82 9.82 0 003.3.77 14.05 14.05 0 003.66-.16 8.48 8.48 0 002.74-.94 5.2 5.2 0 002.69-4.2 7 7 0 00-.7.08z"
          fill="#284a75"
          transform="translate(0 -5.81)"
        />
        <path
          d="M22.87 16h-5.55a.37.37 0 01-.32-.47 3.09 3.09 0 012.18-2.11 3.56 3.56 0 011.86 0 3.13 3.13 0 012.09 2 .53.53 0 01.05.15.38.38 0 01-.31.43zm2.82-1.75a5.81 5.81 0 00-3.81-3.48 6.55 6.55 0 00-3-.14 5.58 5.58 0 00-3.31 1.87 6.81 6.81 0 00-1.39 2.5 8.33 8.33 0 00-.29 3.33 7.78 7.78 0 00.64 2.44A5.77 5.77 0 0020 24.33a6.69 6.69 0 002.79-.63A5.18 5.18 0 0024 23a2.15 2.15 0 00.65-.69 1.06 1.06 0 000-1.17 1.51 1.51 0 00-2.34-.45 3.3 3.3 0 01-3 .72A3 3 0 0117 19.42v-.18a.35.35 0 01.34-.42h7.32a2.06 2.06 0 00.35 0 1.31 1.31 0 001.13-.82 1.44 1.44 0 00.13-.58 7.92 7.92 0 00-.58-3.19zM50.67 17.82a6.72 6.72 0 01-.2 1.27 3.8 3.8 0 01-.64 1.27 3 3 0 01-2.56 1.16 3.09 3.09 0 01-3-1.91 4.75 4.75 0 01-.38-1.32 6.16 6.16 0 01.22-2.73 3.89 3.89 0 01.67-1.19 3.17 3.17 0 013-1 3 3 0 012.51 2.07 4.86 4.86 0 01.38 2.38zm2.79-1.52a7.2 7.2 0 00-1-3.05 5.7 5.7 0 00-3.65-2.6 7.81 7.81 0 00-2-.12 5.68 5.68 0 00-2.49.71 5.91 5.91 0 00-2.56 2.87 8.06 8.06 0 00-.63 4 8.22 8.22 0 00.5 2.35 5.77 5.77 0 003.93 3.69.37.37 0 01.17.13 4.84 4.84 0 001.92.24.74.74 0 00.52-.17 5.65 5.65 0 003.75-2 6.82 6.82 0 001.3-2.41 9.5 9.5 0 00.24-3.64z"
          fill="#2d87f3"
          transform="translate(0 -5.81)"
        />
        <path
          d="M47.27 18.76a1.34 1.34 0 01-1.35-1.35 1.37 1.37 0 112.73 0 1.37 1.37 0 01-1.38 1.35z"
          fill="#284a75"
          transform="translate(0 -5.81)"
        />
      </g>
    </g>
  </svg>
</div>
`;
